File: java/src/main/java/edu/unialfa/java/api/dto/RespostaAlunoDTO.java
package edu.unialfa.java.api.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RespostaAlunoDTO {
    private Long alunoId;
    private Long questaoId;
    private String respostaMarcada;
}

File: correcaogabarito_app/lib/models/RespostaAlunoDTO.dart
class RespostaAlunoDTO {
  final int alunoId;
  final int questaoId;
  final String respostaMarcada;

  RespostaAlunoDTO({
    required this.alunoId,
    required this.questaoId,
    required this.respostaMarcada,
  });

  Map<String, dynamic> toJson() => {
    'alunoId': alunoId,
    'questaoId': questaoId,
    'respostaMarcada': respostaMarcada,
  };
}
